drs_filer/ga4gh/drs/endpoints/registerNewObject.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import logging
+
+logger = logging.getLogger(__name__)
+
+error_respose = {
```

Comment:
Define new error classes for each error (in some module `exceptions.py` or `errors.py`, then create a dictionary of your errors with specific JSON members and handle them through FOCA's generic error handler.